explain/setup/crunchy_bridge/01_auto_explain_check.mdx

```diff
@@ -5,17 +5,17 @@ backlink_href: /docs/explain/setup
 backlink_title: 'Automated EXPLAIN: Setup'
 ---
 
-import Step01InApp from './_01_in_app.mdx'
-import Step01Public from './_01_public.mdx'
+import Step01InAppCB from './_01_in_app.mdx'
+import Step01PublicCB from './_01_public.mdx'
```

Comment:
[note for reviewer] Looks like we have problems when we use the same name (e.g. `Step01InApp`) with different contents. I can see that we're adding this pattern (platform specific name at the end) for other places, so I'm doing it here too. Otherwise, at least locally, I was seeing wrongly cached contents.

Reply:
Uhmm, good point about documenting this. Actually, I've been having a question of how the people are checking if changes that they make "works".
Currently, I'm testing by loading the update in pganalyze-web repo (private) and run `gastby develop` over there but not everyone can do it 🤔 In CONTRIBUTING.md, we could write something around there too.